### DISCUSSION TITLE
iOS app

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#753] [Ideas]
### What feature would you like to see?

I think an iOS app would be awesome, as well as an Android app.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/708_

**Comment by mikalcallahan** [comment]
Could be a nice feature though currently all browsers on iOS run webkit. If Apple opens it up to allow other engines, I can see a lot of benefit.

On a side note: there is [Orion Browser](https://kagi.com/orion/) which is built with webkit but can patch/install extensions from firefox/chrome, and is privacy focused as well (though no open source)

**Comment by agustinroig** [comment]
I'm using Orion Browser, I stopped using Arc for security. I like Zen, but it's not for ios.

**Comment by enslaved2die** [comment]
for the users in the EU this could be amazing for everyone else Im sorry.

**Comment by FelipeCorrales** [comment]
There are currently no good browsers on iOS aside from Orion, I would love to see Zen on iOS!

**Comment by relink2013** [comment]
Hey I know this is old but I found it because I'm looking for the same thing. I wanted to throw my 2 cents in on the topic. 

Yeah, iOS requires all browsers use WebKit, hopefully that changes eventually, but it is what is it. But when people ask for web browser X for iOS, it's not about the underlying Engine, it's about a cohesive and seamless experience across devices. Being able to have all my tabs Open tabs, Spaces, bookmarks, etc, in sync across all devices. 

Firefox on iOS will sync with Zen, but not Spaces, you just get all your tabs in one disorganized list.

I’ve noticed that almost every alternative browser developer has said something similar when people ask for an iOS app. I totally understand—developers are usually focused on the engine, the inner workings, and all the technical details that make their project amazing. But for most users, the technical stuff isn’t really what they care about. What matters is a smooth, feature-rich, and enjoyable experience. The user shouldn’t have to worry about the technical details; they should just be able to use it. I mean, when was the last time anyone thought about the engineering and technical aspects of their hammer? Probably never, because people just want a hammer that can hammer things properly and without breaking.

I hope that came across clearly, and I want to make it clear that I’m not at all criticizing the developers. I’m just sharing my thoughts on the matter. I’m really hoping to see Zen on iOS, not because of the rendering engine, but because I want to keep everything in sync and ready to go, no matter which device I’m using.
